client/app/bundles/KudosApp/actions/tabActions.jsx

```diff
@@ -49,7 +49,8 @@ const fetchPage = (currentTab, offset) => {
       params: {
         limit: 10,
         tab: currentTab,
-        offset: offset
+        cursor_time,
```

Comment:
@trisarahsorus note that with your new mobxify PR #36 we'll want to keep track of the `cursor` (last fetched kudo's `created_at` and `id`) rather than `page` (offset)

Reply:
Please don't delete this branch after you merge so that I can fix mobxifying conflicts without having to merge in master! :,)